dmscripts/helpers/env_helpers.py

```diff
@@ -1,9 +1,9 @@
 def get_api_endpoint_from_stage(stage, app='api'):
-    """Return the full URL of given API or Search API environment.
+    """Return the full URL of given API, Search API or web URL environment.
 
     :param stage: environment name. Can be one of 'preview', 'staging',
                   'production' or 'dev' (aliases: 'local', 'development').
-    :param app: should be either 'api' or 'search-api'
+    :param app: should be either 'api', 'search-api' or 'www'
```

Comment:
We've moved the construction of the URL entirely (including the UTM param) to the Python code so it's all in one place. This should keep the template more readable, considering there are potentially 4 different domains to choose from.